src/rdkafka_admin.c

```diff
@@ -3774,9 +3839,19 @@ rd_kafka_DescribeConfigsResponse_parse(rd_kafka_op_t *rko_req,
                                 rd_list_add(&entry->synonyms, syn_entry);
                         }
 
+                        if (api_version >= 3) {
+                                rd_kafka_buf_read_i8(reply, &config_type);
+                                rd_kafka_buf_read_str(reply, &documentation);
+                                entry->type = config_type;
+                                entry->documentation =
+                                    RD_KAFKAP_STR_DUP(&documentation);
+                        }
```

Comment:
Is this change necessary for KIP-848 changes?

Reply:
No this change was for upgrading the API version. Have removed it.